### ISSUE TITLE
oidc-prompt uses yad with the --splash option

### ISSUE THREAD

**Issue by paulmillar** [indigo-dc/oidc-agent#393]
I configured oidc-agent in such a way that attempting to obtain a token triggered oidc-agent to prompt me to open a URL in a browser.  It did this using the `oidc-prompt` command, which (in turn) used `yad` to show the information.

Unfortunately, the window showing the information was displayed such that when I attempt to select the URL (in order to open it within a browser) the window immediately disappears.  Additionally, the `yad` command never returns, despite the window no longer bring visible.

I've traced the problem to oidc-prompt's use of the `--splash` option in `yad`.  Manually running `yad` with this option recreates the problem, while omitting this option provides a reasonable behaviour.

The `yad` man page says:
>        --splash
>              Open window with "splashscreen" window hints. For details see description of _NET_WM_WINDOW_TYPE_SPLASH in EWMH specification. The behavior  of  dialog  with
>              this option is HIGHLY DEPENDS on settings of your window manager.

I guess this behaviour is the result of my window manager (KDE's kwin) and possibly my choice of its configuration.  However, it looks like the inclusion of the `--splash` option might not be a good idea.

Cheers,
Paul.

**Comment by zachmann** [comment]
In my tests `--splash` brought some nice properties (at least in my window manager), but it probably is best to drop it.

**Comment by paulmillar** [comment]
Yes, unfortunately it looks like the result of including `_NET_WM_WINDOW_TYPE_SPLASH` is just too random.

Just as a thought: you could open a ticket against yad, asking them to support these nice properties you mention without resorting to `--splash`/`_NET_WM_WINDOW_TYPE_SPLASH`.
